### PR TITLE
Functionality placeholders

### PR DESCRIPTION
**Pull request by zackkatz**
- [x] Allow installing and activating inline
- [x] Add check to see if license includes the plugin
- [x] Make install AJAXified
- [x] Ensure refresh after ajax install does not throw away any form data that has been changed

Implements #1980 

💾 [Build file](https://www.dropbox.com/scl/fi/48yocrkix0igb5om8eegg/gravityview-2.25-729b388b1.zip?rlkey=krez7ldrvk6ojxwt4vsi9wfrs&dl=1) (729b388b1).

### CODE REVIEW

**Comment by zackkatz** [comment]
Starting to take shape:

**Comment by rafaehlers** [comment]
@zackkatz I'm unsure if it's included, but please don't forget to include the PDF for GravityView tab.

**Comment by zackkatz** [comment]
@doekenorg Since you have recently worked on this for #1996, I think it makes sense if you were to work on the functionality. Let me know if you have questions.

**Comment by doekenorg** [comment]
@zackkatz I can now probably very quickly add a few extra upsell blocks. Is there a list of all the ones we need, with their location?

**Comment by rafaehlers** [comment]
Tone down a bit and make one tab for the Ratings & Reviews too:

image

**Comment by zackkatz** [comment]
- [ ] Add Ratings & Reviews placeholder

**Comment by mrcasual** [comment]
@zackkatz, I activated Maps and received the following notice:

CleanShot 2024-08-06 at 09 26 10@2x

Is this supposed to happen?

File: assets/js/admin-views.js [hunk 1]
```diff
@@ -3151,4 +3151,69 @@
...
+			if ( $( this ).hasClass( 'is-idle' ) ) {
+				return;
+			}
+
+			$( this ).addClass( 'is-idle' ).html( $spinner );
+
+			const ajaxRoute = $( this ).data( 'action' ) + '_product';
+			const text_domain = $( this ).data( 'text-domain' );
+
+			// Todo: refactor after #1996 is merged.
```
Comment (mrcasual) [inline assets/js/admin-views.js:3177]: @zackkatz or @doekenorg, what kind of refactoring is expected here?

Comment (zackkatz) [reply assets/js/admin-views.js:3177]: Good question! That's one for @doekenorg I believe.

**Pull request merged**